client/src/pages/ForumPage/ForumPageDiscover.js

```diff
@@ -20,7 +23,29 @@ const handleClick = e => {
   console.log("Clicked");
 };
 
-const ForumPageDiscover = props => {
+class ForumPageDiscover extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      topics: []
+    }
+  }
+
+componentDidMount() {
+  axios.get(`${BASE_URL}/topics`)
+    .then(response => {
+      this.setState({ 
+        topics: response.data
+      })
+    })
+    .catch(error => {
+      console.log('Error', error);
+    })
+}
```

Comment:
Can you remove this and use the one you created in the service?

Reply:
Not sure why that showed up on your end, double-checked and removed